solver/src/main/java/org/chocosolver/solver/expression/continuous/relational/BiCReExpression.java

```diff
@@ -73,6 +75,14 @@ public Constraint ibex(double p) {
     }
 
 
+    @Override
+    public Constraint equation() {
+        CArExpression expression = e1.sub(e2);
```

Comment:
Is it always `sub`, whatever the mathematical operator ?

Reply:
Yes it is. Some improvements may be brought at the expense of straightforwardness I suppose.